packages/kuma-gui/src/app/data-planes/views/DataPlaneDetailView.vue

```diff
@@ -38,130 +38,132 @@
+                #body
+              >
+                <XBadge appearance="decorative">
+                  <XAction
+                    :to="{
+                      name: 'zone-cp-detail-view',
+                      params: {
+                        zone: props.data.zone,
+                      },
+                    }"
+                  >
+                    {{ props.data.zone }}
+                  </XAction>
+                </XBadge>
+              </template>
+            </DefinitionCard>
+            <DefinitionCard layout="horizontal">
+              <template #title>
+                Type:
+              </template>
 
-                <DefinitionCard
-                  v-if="props.data.namespace.length > 0"
-                >
-                  <template #title>
-                    Namespace
-                  </template>
+              <template #body>
+                <XBadge appearance="decorative">
+                  {{ t(`data-planes.type.${props.data.dataplaneType}`) }}
+                </XBadge>
+              </template>
+            </DefinitionCard>
 
-                  <template #body>
-                    {{ props.data.namespace }}
-                  </template>
-                </DefinitionCard>
-              </div>
-              <div
-                v-if="props.data.dataplane.networking.gateway"
-                class="columns"
-              >
-                <DefinitionCard>
-                  <template #title>
-                    {{ t('http.api.property.tags') }}
-                  </template>
+            <DefinitionCard
+              v-if="props.data.namespace.length > 0"
+              layout="horizontal"
+            >
+              <template #title>
+                {{ t('http.api.property.namespace') }}:
+              </template>
 
-                  <template #body>
-                    <TagList :tags="props.data.dataplane.networking.gateway.tags" />
-                  </template>
-                </DefinitionCard>
+              <template #body>
+                <XBadge appearance="decorative">
+                  {{ props.data.namespace }}
+                </XBadge>
+              </template>
+            </DefinitionCard>
 
-                <DefinitionCard>
-                  <template #title>
-                    {{ t('http.api.property.address') }}
-                  </template>
+            <DefinitionCard layout="horizontal">
+              <template #title>
+                {{ t('http.api.property.address') }}:
+              </template>
 
-                  <template #body>
-                    <TextWithCopyButton :text="`${props.data.dataplane.networking.address}`" />
-                  </template>
-                </DefinitionCard>
-              </div>
-            </div>
-          </KCard>
+              <template #body>
+                <XBadge appearance="decorative">
+                  <TextWithCopyButton :text="`${props.data.dataplane.networking.address}`" />
```

Comment:
Just a note, if we are editing this PR a bit more may aswell update this to `XCopyButton` while we are here.

Reply:
Yes, absolutely 👍